src/firmware/app/control/wheel_controller.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include "firmware/shared/circular_buffer.h"
+
+// wheel_controller_difference_equation.h
+typedef struct WheelController
+{
+    CircularBuffer_t* previous_command_buffer;
+    CircularBuffer_t* previous_output_sample_buffer;
+    float* command_coefficients;
+    float* output_sample_coefficients;
+    unsigned int num_command_coefficients;
+    unsigned int num_output_sample_coefficients;
+} WheelController_t;
+
+/**
+ * Function creates a WheelController_t with the specified parameters and returns a
+ pointer to it. The wheel_controller_destroy function MUST be called before the
```

Comment:
Comment looks malformed (lines are missing `*` at the start)

Reply:
Fixed 👍